src/commands/hld/reconcile.test.ts

```diff
@@ -741,6 +741,15 @@ describe("normalizedName", () => {
     expect(normalizedName("fabrikam.frontend")).toBe("fabrikam-frontend");
   });
 
+  it("can handle multiple occurences of invalid characters", () => {
+    expect(normalizedName("fabrikam.frontend.foo")).toBe(
+      "fabrikam-frontend-foo"
+    );
+    expect(normalizedName("fabrikam/frontend/foo")).toBe(
+      "fabrikam-frontend-foo"
+    );
+  });
```

Comment:
would be nice to add a test case to test lower casing.

Reply:
it already exists https://github.com/CatalystCode/spk/pull/370/files#diff-948e5cd512781080b25c861e85552153R732